### PR TITLE
docker in production

### PR DESCRIPTION
**Pull request by nzoschke**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _posts/2016-11-11-docker-in-production.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+<!--more-->
+
+## Don’t Do-It-Yourself
+
+Building a custom deployment system with any tech (Docker, Kubernetes, Ansible, Packer, etc.) is a challenge. All the small problems and subtle decisions add up to one big burden on you. 
+
+6 months later, when things aren't totally stable, you get very angry about all the time wasted and blame the tools.
+
+The cost of DIY over the life of a business is. You need at a bare minimum one full-time DevOps Engineer to build and maintain things. But if your business systems are growing, you need a whole team to keep things humming 24/7.
```
Comment (ddollar) [inline _posts/2016-11-11-docker-in-production.md]: "a business is" needs the rest of the sentence.

Comment (soulshake) [inline _posts/2016-11-11-docker-in-production.md]: I think there's a word missing here (is what?)

File: _posts/2016-11-11-docker-in-production.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Use Services, Not Software
+
+One of the articles says Docker is "Banned from the DBA". My Database administrator is the ace teams behind AWS DynamoDB and Heroku Postgres that work non-stop to protect my data.
+
+Configuring an AWS VPC is far simpler than mastering the Docker networking stack. Using ECS is much easier than maintaining your own etcd or Swarm cluster. Using Cloudwatch Logs is cheaper and more reliable than deploying a logging contraption into your cluster.
+
+## A Recipe For Success
+
+If you do need to run Docker in production, this recipe is working well for us at Convox:
```
Comment (soulshake) [inline _posts/2016-11-11-docker-in-production.md]: I suggest saying "run Docker in production **yourself**" to circle it back to your previous "Don't DIY" heading.

File: _posts/2016-11-11-docker-in-production.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+title: Docker In Production for 18 Months
+author: Noah Zoschke
+twitter: nzoschke
+---
+
+There is a wave of Docker retrospectives out there right now. A couple negative:
```
Comment (soulshake) [inline _posts/2016-11-11-docker-in-production.md]: "A couple are negative" maybe?

**Pull request merged**